### PR TITLE
Fix flaky tests

### PR DESCRIPTION
**Pull request by colby-swandale**
### What was the end-user problem that led to this PR?

Currently in travis CI specs are failing intermittantly and makes merging code into master difficult. See: 

* https://github.com/bundler/bundler/pull/6080
* https://github.com/bundler/bundler/pull/6059

### CODE REVIEW

**Comment by olleolleolle** [comment]
@colby-swandale This seemed to clean up the situation immediately. Is this still work-in-progress?

**Comment by colby-swandale** [comment]
yes

File: lib/bundler/rubygems_gem_installer.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "rubygems/installer"
+require "digest/sha2"
```
Comment (segiddins) [inline lib/bundler/rubygems_gem_installer.rb]: instead of this, we should use `Digest(:SHA256)` on line 52?

Comment (colby-swandale) [reply lib/bundler/rubygems_gem_installer.rb]: Yea, that does look better to use and it's theadsafe as well which i think is the the problem we're dealing with.  https://github.com/ruby/ruby/blob/trunk/ext/digest/lib/digest.rb#L96

Comment (segiddins) [reply lib/bundler/rubygems_gem_installer.rb]: ping on making this change?

Comment (colby-swandale) [reply lib/bundler/rubygems_gem_installer.rb]: updated, sorry for the delay 🙇

**Comment by segiddins** [comment]
@bundlerbot r+

**Pull request merged**